frontend/build.sbt

```diff
@@ -24,22 +24,22 @@ organization := "com.gu"
 
 version := "1.0-SNAPSHOT"
 
-scalaVersion := "2.12.13"
+scalaVersion := "2.13.7"
 
 resolvers ++= Seq(
     "Guardian Github Releases" at "https://guardian.github.io/maven/repo-releases",
     Resolver.sonatypeRepo("releases")
 )
 
-sources in (Compile,doc) := Seq.empty
+Compile / doc / sources := Seq.empty
```

Comment:
`in Test` style syntax is loudly deprecated now in favour of the `Test / ` syntax